internal/pkg/local/update.go

```diff
@@ -6,17 +6,64 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/model"
 )
 
-func (m *Manager) UpdatePaths(state model.ObjectState, rename bool) error {
-	object := state.LocalOrRemoteState()
+type PathsUpdater struct {
```

Comment:
2. 

Druha vec, co bolo potrebne fixnut je `config <-> config` vztah.
Doteraz sa generovanie path spoliehalo na to, ze ak idu objekty v poradi `branches, configs, rows`, tak rodic kazdeho objektu uz ma vygenerovanu path.
Pri vztahu `config <-> config` to uz ale neplati, kedze nevieme, ktory config pride ako prvy (aj ked su uz oba isto nacitane).

Preto musime generovat path rekurzivne (podobne ako pri nacitavani manifestu).